File: framework/src/main/java/wulinpeng/com/framework/base/mvp/BasePresenter.java
package wulinpeng.com.framework.base.mvp;

/**
 * @author wulinpeng
 * @datetime: 18/9/22 上午12:51
 * @description:
 */
public class BasePresenter<T extends IBaseContract.IBaseView> {
    protected T mRootView;

    public BasePresenter(T mRootView) {
        this.mRootView = mRootView;
    }
}

File: framework/src/main/java/wulinpeng/com/framework/base/mvp/BasePresenter.kt
package wulinpeng.com.framework.base.mvp

/**
 *
 * @author wulinpeng
 * @datetime: 2019/4/6 8:47 PM
 * @description:
 */
open class BasePresenter<T: IBaseContract.IBaseView>(@JvmField var mRootView: T) {
}